app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,14 @@
+<hr>
+<nav class="navbar navbar-buttom" role="navigation">
+  <div class="container">
+    <small>
+      The <%= link_to t(".title_railstt"), "http://www.railstutorial.org" %>
+      by <%= link_to t(".author"), "http://www.michaelhartl.com" %>
+    </small>
```

Comment:
Sao mà vẫn để lỗi I18n vầy em `The` `by`

em nên dùng 1 message thôi em nhé. Như bên dưới:
<%= t(".small_content", first_links: link_to(t(".first_href"), "http://www.railstutorial.org/"), second_links: link_to(t(".second_href"), "http://www.michaelhartl.com/")).html_safe%>

Và file en em chỉ cần định nghĩa:
footer:
       small_content: "The %{first_links} by %{second_links}"
      first_href: "Ruby on Rails Tutorial"
      second_href: "Michael Hartl"